src/custom/state/orders/consts.ts

```diff
@@ -7,3 +8,6 @@ export const ContractDeploymentBlocks: Partial<Record<ChainId, number>> = {
 }
 
 export const OPERATOR_API_POLL_INTERVAL = 10000 // in ms
+export const PENDING_ORDERS_PRICE_CHECK_POLL_INTERVAL = 30000 // in ms
```

Comment:
Is it to bad to do it every 10s? 

There's a bunch of stuff (like this one) that I would love to change it to logic we trigger on every new block instead of at regular intervals. 

10s is just a rudimentary way to make sure we update on every block.

Reply:
Ignore me if 10s will risk hitting time limit of some sort. I'm assuming they allow several per second, if it's not the case 30s as u suggest is good enough

Reply:
I did not face any rate limiting issues.
I'm just being conservative in having stuff running on the brackground updating too often, to avoid making the page slow(er than usual) and overloading the query endpoints with too many requests.

I can change it to 10s if you think that's fine.

Reply:
It's alright. Make sense